src/test/java/com/hyperlocal/server/ShopControllerTest.java

```diff
@@ -64,20 +70,14 @@ public void shouldGetShopsByMerchantID() throws Exception {
     assertThat(controller).isNotNull();
     String merchantID = "1000000000000";
 
-    RowData shopRecord = new FakeRowData(
-      "ShopID", 1L, 
```

Comment:
Might be a good idea to preserve this indentation as it make the `ColumnName => ColumnValue` mapping readable

Reply:
Done. Thanks a lot!!!